README.md

```diff
@@ -2332,6 +2332,10 @@ Pass `-help` to VictoriaMetrics in order to see the list of supported command-li
      Supports an array of values separated by comma or specified via multiple flags.
   -tlsKeyFile string
      Path to file with TLS key if -tls is set. The provided key file is automatically re-read every second, so it can be dynamically updated
+  -tlsMinVersion string
```

Comment:
Could you please add a note with accepted values for these flags?
It might require checking code to understand expected format.

Reply:
added